backend/controllers/download_accessions_controller.rb

```diff
@@ -0,0 +1,104 @@
+require 'csv'
+
+class ArchivesSpaceService < Sinatra::Base
+
+  Endpoint.get('/repositories/:repo_id/download_accessions')
+    .description("Download Accessions as CSV")
+    .params(["q", String, "A search query string",
+             :optional => true],
+            ["aq", JSONModel(:advanced_query), "A json string containing the advanced query",
+             :optional => true],
+            ["filter_term", [String], "A json string containing the term/value pairs to be applied as filters.  Of the form: {\"fieldname\": \"fieldvalue\"}.",
+             :optional => true],
+            ["repo_id", :repo_id])
```

Comment:
To accept all search params these could be replaced with `*BASE_SEARCH_PARAMS,`.  This would also allow the `sort` param to come through.